tests/test_auth/test_admin_access_endpoints_integration.py

```
@@ -0,0 +1,640 @@
+            if org_id:
+                conn.execute("DELETE FROM organizations WHERE id = %s", (org_id,))
+            conn.execute("DELETE FROM users WHERE email = ANY(%s)",
+                         ([admin_email, owner_email, member_email],))
+            conn.commit()
+
+
+def test_assign_member_unknown_user_returns_400():
+    """A nonexistent user_id is a bad request, not a 500 (FK violation → 400)."""
+    init_invitations_db()
+    repo, orgs = UserRepository(), OrgRepository()
+    admin_email, token = _admin_token()
```

Ruff (0.15.20)

[warning] Unpacked variable `admin_email` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)